scripts/checks/run-windows-native-installer-qualification.ps1

```
@@ -0,0 +1,715 @@
+
+    return [pscustomobject]@{
+        exitCode = $exitCode
+        processId = $processId
+        sideEffectObserved = Test-Path -LiteralPath $SentinelPath -PathType Leaf
+        startRejected = $null -ne $startError
+    }
+}
+
+function Start-ProcessStartAudit {
+    $sourceIdentifier = 'NemoClawNativeInstaller-' + [guid]::NewGuid().ToString('N')
+    $subscription = Register-WmiEvent -Class Win32_ProcessStartTrace -SourceIdentifier $sourceIdentifier
```

PSScriptAnalyzer (1.25.0)

[warning] File 'run-windows-native-installer-qualification.ps1' uses WMI cmdlet. For PowerShell 3.0 and above, use CIM cmdlet which perform the same tasks as the WMI cmdlets. The CIM cmdlets comply with WS-Management (WSMan) standards and with the Common Information Model (CIM) standard, which enables the cmdlets to use the same techniques to manage Windows computers and those running other operating systems.

(PSAvoidUsingWMICmdlet)